database/migrations/2024_03_23_072523_create_order_status_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('order_status', function (Blueprint $table) {
+            $table->id();
+            $table->string('status')->unique()->comment('1:Waiting for packing -> 2:Waiting for acceptance -> (6:Cancellation -> 7:Refund completed ->) 3:Acceptance complete -> 4:In transit -> 5:Delivery completed');
```

Comment:
use int for status

Reply:
Sorry this is a kind of master table , so this one must be not changable. I will modify this again.